backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Data/ExpectedResults/SingleRight/CreateDelegation/a3-app2.json

```diff
@@ -22,7 +22,7 @@
     {
       "resource": [
         {
-          "id": "urn:altinn:applicationid",
+          "id": "urn:altinn:app",
```

Comment:
All services are now green on both ChooseServicePage and ChooseRightsPage. So there's probably some stuff that needs to be updated there as well.

Reply:
Fixed! 😊